lib/bigcommerce/connection.rb

```diff
@@ -34,24 +34,24 @@ def api_key=(api_key)
       @configuration[:api_key] = api_key
     end
 
-    def verify_peer=(verify)
+    def verify_ssl=(verify)
       @configuration[:verify_ssl] = verify
     end
 
     def ssl_ca_file=(path)
-      @configuration.ssl_ca_file = path
+      @configuration[:ssl_ca_file] = path
     end
 
-    def ssl_client_key=(path,passphrase=nil)
-      if passphrase.nil?
-        @configuration.ssl_client_key = OpenSSL::PKey::RSA.new(File.read(path))
+    def ssl_client_key=(options)
```

Comment:
`ssl_client_key` field takes max two values in the API configuration `path` and `passphrase`(optional). This setter method `ssl_client_key=` is called dynamically in the constructor of `Connection`.  If we keep the method signature same then that just means we need to add checks and stuff in constructor and break the supplied hash and pass the values to this setter method explicitly. It will add a little bit complexity in the code. Let me know your thoughts on it so I can proceed to make this change.